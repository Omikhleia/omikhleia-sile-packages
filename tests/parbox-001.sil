\begin{document}
\neverindent
\nofolios
\script[src=packages/parbox]
\script[src=packages/lorem]
% Test case 001 = Real basic/minimal parbox test. If this one fails, we broke something deep.

\parbox[width=33%fw]{\lorem[words=100]}

\end{document}