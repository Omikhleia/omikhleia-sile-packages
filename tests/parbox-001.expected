Set paper size 	595.275597	841.8897729
Begin page
Mx 	29.7638
My 	49.6628
Set font 	Gentium Plus;10;400;;normal;;LTR
T	79 82 85 72 80 w=24.3506	(lorem)
Mx 	57.2206
T	76 83 86 88 80 w=25.2002	(ipsum)
Mx 	85.5271
T	71 82 79 82 85 w=21.9287	(dolor)
Mx 	110.5621
T	86 76 87 w=10.0146	(sit)
Mx 	123.6830
T	68 80 72 87 w=20.6836	(amet)
Mx 	147.4729
T	70 82 81 86 72 87 72 87 88 85 w=44.1602	(consetetur)
Mx 	194.7393
T	86 68 w=8.4521	(sa)
Mx 	203.1915
T	16 w=3.3691	(-)
Mx 	29.7638
My 	61.6628
T	71 76 83 w=13.2080	(dip)
Mx 	42.9718
T	86 70 76 81 74 w=21.3379	(scing)
Mx 	66.2182
T	72 79 76 87 85 w=17.4414	(elitr)
Mx 	85.5681
T	86 72 71 w=13.6816	(sed)
Mx 	101.1583
T	71 76 68 80 w=20.5322	(diam)
Mx 	123.5991
T	81 82 81 88 80 92 w=34.3262	(nonumy)
Mx 	159.8338
T	72 76 85 80 82 71 w=29.5508	(eirmod)
Mx 	191.2931
T	87 72 80 w=16.0938	(tem)
Mx 	207.3868
T	16 w=3.3691	(-)
Mx 	29.7638
My 	73.6628
T	83 82 85 w=14.2871	(por)
Mx 	46.4595
T	76 81 89 76 71 88 81 87 w=35.3271	(invidunt)
Mx 	84.1953
T	88 87 w=8.7402	(ut)
Mx 	95.3441
T	79 68 69 82 85 72 w=25.9863	(labore)
Mx 	123.7391
T	72 87 w=8.0615	(et)
Mx 	134.2092
T	71 82 79 82 85 72 w=26.5479	(dolore)
Mx 	163.1657
T	80 68 74 81 68 w=27.6074	(magna)
Mx 	193.1817
T	68 79 76 w=10.0098	(ali)
Mx 	203.1915
T	16 w=3.3691	(-)
Mx 	29.7638
My 	85.6628
T	84 88 92 68 80 w=28.0029	(quyam)
Mx 	60.8080
T	72 85 68 87 w=16.6113	(erat)
Mx 	80.4606
T	86 72 71 w=13.6816	(sed)
Mx 	97.1836
T	71 76 68 80 w=20.5322	(diam)
Mx 	120.7571
T	89 82 79 88 83 87 88 68 w=36.5967	(voluptua)
Mx 	160.3950
T	68 87 w=8.0322	(at)
Mx 	171.4686
T	89 72 85 82 w=18.5400	(vero)
Mx 	193.0499
T	72 82 86 w=13.5107	(eos)
Mx 	29.7638
My 	97.6628
T	72 87 w=8.0615	(et)
Mx 	41.5985
T	68 70 70 88 86 68 80 w=35.1123	(accusam)
Mx 	80.4840
T	72 87 w=8.0615	(et)
Mx 	92.3187
T	77 88 86 87 82 w=20.2197	(justo)
Mx 	116.3116
T	71 88 82 w=15.5273	(duo)
Mx 	135.6122
T	71 82 79 82 85 72 86 w=30.4102	(dolores)
Mx 	169.7955
T	72 87 w=8.0615	(et)
Mx 	181.6302
T	72 68 w=9.2090	(ea)
Mx 	194.6124
T	85 72 w=8.5791	(re)
Mx 	203.1915
T	16 w=3.3691	(-)
Mx 	29.7638
My 	109.6628
T	69 88 80 w=18.4082	(bum)
Mx 	52.2924
T	86 87 72 87 w=15.3662	(stet)
Mx 	71.7790
T	70 79 76 87 68 w=17.8223	(clita)
Mx 	93.7216
T	78 68 86 71 w=18.5303	(kasd)
Mx 	116.3723
T	74 88 69 72 85 74 85 72 81 w=42.8076	(gubergren)
Mx 	163.3003
T	81 82 w=10.5469	(no)
Mx 	177.9676
T	86 72 68 w=13.0713	(sea)
Mx 	195.1593
T	87 68 w=8.0322	(ta)
Mx 	203.1915
T	16 w=3.3691	(-)
Mx 	29.7638
My 	121.6628
T	78 76 w=7.5879	(ki)
Mx 	37.3517
T	80 68 87 68 w=20.6543	(mata)
Mx 	62.0384
T	86 68 81 70 87 88 86 w=30.9424	(sanctus)
Mx 	97.0131
T	72 86 87 w=11.9238	(est)
Mx 	112.9693
T	79 82 85 72 80 w=24.3506	(lorem)
Mx 	141.3523
T	76 83 86 88 80 w=25.2002	(ipsum)
Mx 	170.5849
T	71 82 79 82 85 w=21.9287	(dolor)
Mx 	196.5460
T	86 76 87 w=10.0146	(sit)
Mx 	29.7638
My 	133.6628
T	68 80 72 87 w=20.6836	(amet)
Mx 	53.6318
T	79 82 85 72 80 w=24.3506	(lorem)
Mx 	81.1668
T	76 83 86 88 80 w=25.2002	(ipsum)
Mx 	109.5514
T	71 82 79 82 85 w=21.9287	(dolor)
Mx 	134.6645
T	86 76 87 w=10.0146	(sit)
Mx 	147.8635
T	68 80 72 87 w=20.6836	(amet)
Mx 	171.7315
T	70 82 81 w=14.9170	(con)
Mx 	186.6485
T	86 72 w=8.4814	(se)
Mx 	195.1300
T	87 72 w=8.0615	(te)
Mx 	203.1915
T	16 w=3.3691	(-)
Mx 	29.7638
My 	145.6628
T	87 88 85 w=12.7002	(tur)
Mx 	45.8738
T	86 68 71 76 83 86 70 76 81 74 w=42.9980	(sadipscing)
Mx 	92.2817
T	72 79 76 87 85 w=17.4414	(elitr)
Mx 	113.1329
T	86 72 71 w=13.6816	(sed)
Mx 	130.2244
T	71 76 68 80 w=20.5322	(diam)
Mx 	154.1664
T	81 82 81 88 80 92 w=34.3262	(nonumy)
Mx 	191.9024
T	72 76 85 w=11.2891	(eir)
Mx 	203.1915
T	16 w=3.3691	(-)
Mx 	29.7638
My 	157.6628
T	80 82 71 w=18.2617	(mod)
Mx 	51.9407
T	87 72 80 83 82 85 w=30.3809	(tempor)
Mx 	86.2368
T	76 81 89 76 71 88 81 87 w=35.3271	(invidunt)
Mx 	125.4791
T	88 87 w=8.7402	(ut)
Mx 	138.1345
T	79 68 69 82 85 72 w=25.9863	(labore)
Mx 	168.0361
T	72 87 w=8.0615	(et)
Mx 	180.0128
T	71 82 79 82 85 72 w=26.5479	(dolore)
Mx 	29.7638
My 	169.6628
T	80 68 74 81 68 w=27.6074	(magna)
Mx 	59.9916
T	68 79 76 84 88 92 68 80 w=38.0127	(aliquyam)
Mx 	100.6248
T	72 85 68 87 w=16.6113	(erat)
Mx 	119.8565
T	86 72 71 w=13.6816	(sed)
Mx 	136.1586
T	71 76 68 80 w=20.5322	(diam)
Mx 	159.3113
T	89 82 79 88 83 87 88 68 w=36.5967	(voluptua)
Mx 	198.5284
T	68 87 w=8.0322	(at)
Mx 	29.7638
My 	181.6628
T	89 72 85 82 w=18.5400	(vero)
Mx 	50.7153
T	72 82 86 w=13.5107	(eos)
Mx 	66.6376
T	72 87 w=8.0615	(et)
Mx 	77.1106
T	68 70 70 88 86 68 80 w=35.1123	(accusam)
Mx 	114.6344
T	72 87 w=8.0615	(et)
Mx 	125.1074
T	77 88 86 87 82 w=20.2197	(justo)
Mx 	147.7386
T	71 88 82 w=15.5273	(duo)
Mx 	165.6775
T	71 82 79 82 85 72 86 w=30.4102	(dolores)
Mx 	198.4991
T	72 87 w=8.0615	(et)
Mx 	29.7638
My 	193.6628
T	72 68 w=9.2090	(ea)
Mx 	42.1808
T	85 72 69 88 80 w=26.9873	(rebum)
Mx 	72.3761
T	86 87 72 87 w=15.3662	(stet)
Mx 	90.9503
T	70 79 76 87 68 w=17.8223	(clita)
Mx 	111.9806
T	78 68 86 71 w=18.5303	(kasd)
Mx 	133.7188
T	74 88 69 72 85 74 85 72 81 w=42.8076	(gubergren)
Mx 	179.7345
T	81 82 w=10.5469	(no)
Mx 	193.4893
T	86 72 68 w=13.0713	(sea)
Mx 	29.7638
My 	205.6628
T	87 68 78 76 80 68 87 68 w=36.2744	(takimata)
Mx 	68.7319
T	86 68 81 70 87 88 86 w=30.9424	(sanctus)
Mx 	102.3679
T	72 86 87 w=11.9238	(est)
Mx 	116.9854
T	79 82 85 72 80 w=24.3506	(lorem)
Mx 	144.0297
T	76 83 86 88 80 w=25.2002	(ipsum)
Mx 	171.9236
T	71 82 79 82 85 w=21.9287	(dolor)
Mx 	196.5460
T	86 76 87 w=10.0146	(sit)
Mx 	29.7638
My 	217.6628
T	68 80 72 87 w=20.6836	(amet)
End page
Finish
